src/agent/onefuzz-telemetry/src/lib.rs

```diff
@@ -124,6 +147,37 @@ pub enum EventData {
     ToolName(String),
     Region(String),
     Role(Role),
+    InputsFuzzed(u64),
+    SatConstraints(u64),
+    UnsatConstraints(u64),
+    AverageVarsPerConstraint(u64),
+    MaxConstraintVars(u64),
+    AverageSymexTime(f64),
+    MaxSymexTime(u64),
+    AverageSolvingTime(f64),
+    MaxSolvingTime(u64),
+    UniqueCodeLocationCount(u64),
+    AverageInstructionsExecuted(f64),
+    MaxInstructionsExecuted(u64),
+    AverageTaintedInstructions(f64),
+    MaxTaintedInstructions(u64),
+    AverageMemoryTaintedInstructions(f64),
+    MaxMemoryTaintedInstructions(u64),
+    AveragePathLength(f64),
+    MaxPathLength(u64),
+    DiverganceRate(f64),
```

Comment:
Throughout: s/Divergance/Divergence

Reply:
Did you mean to push some changes?

Reply:
Thanks to my internet outage, I pushed from my laptop and then clicked resolve from my phone.

Thanks flaky internet!